### ISSUE TITLE
What Kind of "Getting Started" Content Should Exist?

### ISSUE THREAD

**Issue by bnb** [nodejs/website-redesign#39]
In today's meeting we discussed having multiple paths for "Getting Started" with Node.js. My own personal experience is that Node.js is never _just_ Node.js, but it's always _Node.js and (x)_.

Per some of the discussions we had today, we'd love to get input on what kinds of Getting Started Node.js content _you_ would like to see.

**Comment by JonahMoses** [comment]
To get this kick started, here are a couple suggestions:
- Express server to serve up simple APIs for front end focused web apps
- Node.js on Raspberry Pi

**Comment by bnb** [comment]
From my own personal experience, here's a few:

* Node.js and Front-end
	* Can there be variants of this? For example, Node.js and React, Node.js and Angular, Node.js and Ember, etc.
* Node.js and Server-side JavaScript
	* Node.js and (Your Favorite Web Framework)
	* Node.js and APIs
* Node.js and DevOps
	* Node.js and Docker
	* Node.js and APMs

**Comment by maddhruv** [comment]
As @JonahMoses suggested we can add the **IoT** part too on the box
- Node.js and IoT
     - johnny-five and others available
- Node.js and Desktop Applications (maybe)
     - electron
- Node.js and Testing
- Node.js and Automation

(append to @bnb list :stuck_out_tongue_winking_eye:  )

**Comment by milapbhojak** [comment]
@bnb Really good question and interesting design problem. What if we discuss possible use cases of ended up clicking on “Getting started”? I am sure after knowing all use cases we can easily come up with user goals and organised content. Which would leads us to create better UX. I am studying “Getting started” patterns from other open source projects. I would update it with possible use case soon. 

@amiller-gh Don’t you think we need **user persona** also before we start working on **“Voice and tone guidelines #13”** Let me know what do you think. I could start working on it. 

NodeJS is one of the largest open source project. I am sure we would get good response from community if we just do one quick user research which would help us to create better IA. Let me know what do you think about conducting user research.

**Comment by codeekage** [comment]
Hello, sorry I missed the last meetings was a little bit since then. 

I don't know if it would make sense to work with @nodejs/user-feedback  team to get more contents on what to add to as a getting started a course.

We could also do a poll on slack, gitter, and twitter to see what folks mostly use node.js for? 

Then probably more content can be added based on requests.

**Comment by dkundel** [comment]
Hey folks! 

While we are talking about "Getting Started" I have the feeling that one thing that is missing is a very fundamental: **What is Node.js**

Basically something that covers what Node.js is about in a beginner friendly manner and also inspires what is possible with it. That could segway into the suggested topics. 

I see a lot of beginners being confused even with basics like: What's the difference between Node.js and running JavaScript in the browser. Especially if they come from a different programming language like Ruby.

Cheers,
Dominik

**Comment by Belar** [comment]
Hello,
adding to Dominik's (@dkundel) point of where to begin, I remember 2 cases (pretty recent) which are good examples, points of view. 

One is [article](http://bradfrost.com/blog/post/my-struggle-to-learn-react/) by Brad Frost on his experience with learning React, the other is [issue](https://github.com/hapijs/hapijs.com/issues/264) submitted to Hapi website repo about getting started page. Both [at some point] touch on the topic of missing steps and "explaining with assumptions", which led to confusion of users.

For "Node as API" path, getting started example could be bare Node (no framework); `http`, `url`, maybe `fs`. This would be enough to explain built-in packages, creating a "server", handling requests and serving content. Knowledge which should translate well to frameworks (Express, Koa, Hapi etc., introduced later), making their "getting started" content (and various tutorials, educational content in general, across the web) easier to pick up; this approach should also help to avoid "jQuery = JavaScript"-like learning path.

**Comment by amiller-gh** [comment]
Closing in favor of #60 lets continue discussion over there  ❤️
